.github/workflows/release.yml

```diff
@@ -106,3 +141,34 @@ jobs:
         with:
           env-name: ${{ needs.pre_job.outputs.env_name }}
           version: ${{ needs.pre_job.outputs.version }}
+
+  enable_functions:
+    name: "Set function ${{ matrix.function }} enabled"
+    # Run AFTER deploy job
+    needs:
+      - pre_job
+      - deploy_release
+    # Always run if disable_functions job was triggered
+    if: |
+      always() &&
+      (needs.pre_job.outputs.has_router_change == 'true' ||
+      needs.pre_job.outputs.has_react_change == 'true')
```

Comment:
instead of using a proxy for whether the disable functions job was run, maybe we should set a variable/output when we do so?